README.md

```diff
@@ -54,3 +54,24 @@ For documentation see the
 developer version or the most recent released
 <a href="https://scitools-iris.readthedocs.io/en/stable/">stable</a> version.
 </p>
+
+## [#ShowYourStripes](https://showyourstripes.info/s/globe)
+
+<h4 align="center">
+  <a href="https://showyourstripes.info/s/globe">
+    <img src="https://raw.githubusercontent.com/ed-hawkins/show-your-stripes/master/2021/GLOBE---1850-2021-MO.png"
```

Comment:
We can update this as and when new yearly (?) images are published @ https://github.com/ed-hawkins/show-your-stripes

Reply:
Having recently written out the steps it takes to do a release (#5134), I'd rather leave this out for the sake of brevity!

This still has the appropriate impact even if it isn't the latest, and whenever we have a moment it will be easy to update.